plugins/dashboard-object-viewer/src/js/src/DashboardPlugin/DashboardPlugin.tsx

```diff
@@ -28,7 +28,10 @@ export function DashboardPlugin({
       widget: VariableDefinition;
     }) => {
       const { id: widgetId, name, type } = widget;
-      if (type === dh.VariableType.TABLE || type === dh.VariableType.FIGURE) {
+      if (
+        type === 'dh.VariableType.TABLE' ||
+        type === 'dh.VariableType.FIGURE'
+      ) {
```

Comment:
These types are actually defined in the JS API. We _should_ be getting an instance of the API using `dh = useApi()`, like in ObjectIcon: https://github.com/deephaven/web-client-ui/blob/7ff0e53ac41887fa22f2591b5df2439ac9984397/packages/console/src/common/ObjectIcon.tsx#L13
But that would require adding another dependency on jsapi-bootstrap package, and this dashboard-object-viewer can likely get deprecated anyway, so just hardcode the constants as they're defined in the JS API: https://github.com/deephaven/deephaven-core/blob/e7f27cb80b43f2a93dea2c010198dc7e945484cb/web/client-api/src/main/java/io/deephaven/web/client/api/console/JsVariableType.java#L15
```suggestion
      if (type === 'Table' || type === 'Figure') {
```